src/card.php

```diff
@@ -2,6 +2,45 @@
 
 declare(strict_types=1);
 
+/**
+ * Set headers and echo response based on type
+ *
+ * @param string|array $output
+ * @return void
+ */
+function renderOutput(string|array $output): void
+{
+    $requestedType = $_REQUEST['type'] ?? 'svg';
+
+    $card = gettype($output) === "string" ? generateErrorCard($output) : generateCard($output);
+
+    if ($requestedType === "json") {
+        $data = gettype($output) === "string" ? array("error" => $output) : $output;
+        echoAsJson($data);
+        exit;
+    }
+
+    if ($requestedType === "png") {
+        echoAsPng($card);
+        exit;
+    }
+    
+    echoAsSvg($card);
+    exit;
+}
+
+/**
+ * @param array|string $data
+ */
+function echoAsJson(array|string $data): void
+{
+// set content type to JSON
+    header('Content-Type: application/json');
+    // echo JSON error message
```

Comment:
It's not only for error message

```suggestion
    // echo JSON data
```